### ISSUE TITLE
Suggestion: Recordings navigation to be not on top of the video

### ISSUE THREAD

**Issue by Thorvarium** [blakeblackshear/frigate-hass-addons#116]
Can we have a setting to make the overlay be on a collapsible sidebar instead of overlay?
At least for me, I find it very obnoxious to have this overlay covering half of the video, also the video is so big that makes my screen have scrollbar. Making it into a sidebar would make the video smaller and more pleasant to check. You always have the option to open it full screen, so I think that it would be the optimal default experience.

![image](https://github.com/blakeblackshear/frigate-hass-addons/assets/6555325/4fd889e1-d35f-4362-97ad-ed70810a4cfb)
![image](https://github.com/blakeblackshear/frigate-hass-addons/assets/6555325/f9bdbe69-c31a-4efd-bc83-93be2a0a49c2)

**Comment by NickM-27** [comment]
It's already collapsible, just click the handle in the middle of the drawer

**Comment by Thorvarium** [comment]
> It's already collapsible, just click the handle in the middle of the drawer

Right, but If I am scrolling and watching a series of videos I need to collapse/collapse between every single video. Having as a sidebar is a much better UX

**Comment by NickM-27** [comment]
Sure, we have much different UI for an upcoming player. For now, the playlist element doesn't support being added as a sidebar.
